bluepyopt/ephys/models.py

```diff
@@ -315,20 +316,42 @@ def create_hoc(self, param_values,
                 replace_axon += '\n'
                 replace_axon += morph_modifier_hoc
 
-        ret = create_hoc.create_hoc(mechs=self.mechanisms,
-                                    parameters=self.params.values(),
-                                    morphology=morphology,
-                                    ignored_globals=ignored_globals,
-                                    replace_axon=replace_axon,
-                                    template_name=template_name,
-                                    template_filename=template,
-                                    template_dir=template_dir,
-                                    disable_banner=disable_banner)
+        ret = sim_desc_creator(mechs=self.mechanisms,
+                                 parameters=self.params.values(),
+                                 morphology=morphology,
+                                 ignored_globals=ignored_globals,
+                                 replace_axon=replace_axon,
+                                 template_name=template_name,
+                                 template_filename=template,
+                                 template_dir=template_dir,
+                                 disable_banner=disable_banner)
```

Comment:
Indentation seems to be a tiny bit off.

Reply:
Fixed in https://github.com/BlueBrain/BluePyOpt/pull/393/commits/d2f2f42e05e4a74caaaaa19d44453d42c5b0c947